### ISSUE TITLE
Incoming Transactions Don't Show Up On MetaMask...

### ISSUE THREAD

**Issue by Dycedlp** [paritytech/parity-ethereum#9966]
Hey, 
I'm running a POA, zero gas required private parity-ethereum blockchain.
The incoming transactions don't show on metamask, the outgoing transactions work perfectly, goes pending to confirmed.  I'm wondering if theres any flags I'm missing for this to work.
---------
Parity Ethereum version: 1.27.2
Operating system: Linux
Installation: ubuntu 16
Fully synchronized: yes
Network: ethereum (private)
Restarted: yes
--warp --tracing=on --fat-db=on --pruning=archive --no-discovery --config /ethereum/poa/node1.toml --min-gas-price 0 --ws-interface all --ws-origins all --ws-hosts all
node.toml:
[parity]
chain = "privatepoa-spec.json"
base_path = "/ethereum/poa/node1/"
[network]
port = 30301
[rpc]
port = 8547
apis = ["web3", "eth", "net", "personal", "parity", "parity_set", "traces", "rpc", "parity_accounts"]
cors = ["null","localhost:3000","localhost:3001","http://remix.ethereum.org","chrome-search://ae62fc16-c916-4c24-9a36-97f5a850f25c","all","https://www.myetherwallet.com/"]
interface = "all" #external access to rpc
hosts = ["all"]
[ui]
port = 8181
[websockets]
disable = false
port = 8456
[ipc]
disable = false
[account]
unlock = ["0x0496632f490cb4fbe6559d202120a6fd332442e7"]
password = ["node.pwds"]
[mining]
engine_signer = "0x0496632f490cb4fbe6559d202120a6fd332442e7"
reseal_on_txs = "all"

**Comment by joshua-mir** [comment]
Hey @Dycedlp I don't have metamask available right in front of me atm, but does metamask usually show incoming transactions in your transaction history? I was under the impression it only shows the local history (made in metamask) of outgoing transactions.

**Comment by Dycedlp** [comment]
Yes your correct.  The real problem is when I'm running metamask locally, on the same computer as my private blockchain, when I send tokens theres a popup window that says 'Transaction # Confirmed, view on Etherscan', seems perfect.  But when I run metamask from a non-local comp, the transaction still gets sent, but there is no popup confirmation window that says 'Transaction Confirmed'.  I know this is causing problems else where.  What would allow the popup anywhere? @joshua-mir

**Comment by joshua-mir** [comment]
This doesn't seem to be a universal issue, other users have it working (ie, https://github.com/MetaMask/metamask-extension/issues/5631) you try updating to a stable release of parity and see if it still doesn't show up? In any case this does seem like a metamask based issue, you might want to check metamask's logs to see if they are saying anything useful...

**Comment by Dycedlp** [comment]
I upgraded to 2.2.1, still no metamask popup confirmation window on a non-local machine.  Thanks for the suggestions though.
